app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,13 @@
+module AdminHelper
+  def certificate_component_url(component, component_id)
```

Comment:
`def certificate_component_url(component, component_id)`
There first parameter would be better as component_type